jupyter_ros_server/static/settings.json

```diff
@@ -1 +1 @@
-{"env": "", "master": ""}
\ No newline at end of file
+{"env": "", "master": "", "workspaces": "/Users/kunkuma/ros_x_mac/dev_ws:/Users/kunkuma/ros_x_mac/igor_ws"}
```

Comment:
It would be better to default to an empty string here.

Reply:
Thanks for mentioning it, removed it in the latest commit.